units/units.hpp

```diff
@@ -1788,6 +1788,20 @@ inline fixed_precise_measurement sqrt(const fixed_precise_measurement& meas)
     return root(meas, 2);
 }
 
+UNITS_EXPORT int setUnitsDomain(int newDomain);
+
+/** specify a domain to use in unit translation for some ambiguous units*/
+namespace domains {
+    constexpr std::uint32_t defaultDomain{0U};
+    constexpr std::uint32_t ucum{1U};
+    constexpr std::uint32_t cooking{2U};
+    constexpr std::uint32_t astronomy{4U};
+    constexpr std::uint32_t nuclear{8U};
+    constexpr std::uint32_t surveying{9U};
```

Comment:
So surveying is `ucum` and `nuclear`?

Reply:
guess I need to double check the bit sequences again